server/game/cards/events/03/healingexpertise.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../../drawcard.js');
+
+class HealingExpertise extends DrawCard {
+    canPlay() {
+        return false;
+    }
+
+    setupCardAbilities(ability) {
+        this.interrupt({
+            canCancel: true,
+            when: {
+                onCharactersKilled: event => event.allowSave
+            },
+            location: 'hand',
+            cost: ability.costs.kneel(card => card.hasTrait('Maester') && card.getType() === 'character'),
+            target: {
+                activePromptTitle: 'Select character to save',
+                cardCondition: (card, context) => context.event.cards.includes(card) && !card.hasTrait('Army') && card.controller === this.controller
+            },
+            handler: context => {
+                context.event.saveCard(context.target);                
+                this.game.addMessage('{0} plays {1} to kneel {2} to save {3}', 
```

Comment:
It is on purpose, to distinguish between cost and effect, although it doesn't really look the best...